src/lib/util.ts

```diff
@@ -382,9 +384,21 @@ export async function sendHolidayMessage(holidayName: "easter" | "halloween" | "
 	} else await adminBots.send(`For some reason there is nothing in the ${holidayNameFormatted} announcement so I can't send it. Fix your code bro.`)
 }
 
+export const botLocales = readdirSync("./strings")
+
 export function transformDiscordLocale(discordLocale: string): string {
 	// We support all of Discord's languages so we only need to do this
-	return discordLocale.replace("-", "_")
+	discordLocale = discordLocale.replace("-", "_")
+	if (botLocales.includes(discordLocale)) return discordLocale
+	return "en"
+}
+
+export function transformBotLocale(botLocale: string): LocaleString | null {
+	const isLocale = (locale: string): locale is LocaleString => Object.values(Locale).includes(locale as Locale)
+
+	botLocale = botLocale.replace("_", "-")
```

Comment:
`en` won't be converted to `en-US` or `en-GB`

Reply:
That’s intentional, if you don’t pass localizations for a language it uses the default ones